specification/servicefabric/resource-manager/Microsoft.ServiceFabric/2016-09-01/servicefabric.json

```diff
@@ -314,10 +314,10 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.ServiceFabric/locations/{location}/environments/{environment}/clusterVersions": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ServiceFabric/locations/{location}/clusterVersions": {
       "get": {
         "tags": [
-          "ClusterVersionsListOperation"
+          "ClusterVersion"
```

Comment:
It looks like you're making breaking changes:
- Removing an enum
- Modifying arguments for several get commands

I didn't mean this line specifically since there are multiple changes, but if there are breaking changes we need to release the spec as a different version. Unless you're intentionally making these breaking changes because the spec is incorrect for the corresponding service fabric runtime version.